docs/src/userguide/wrapping_CPlusPlus.rst

```diff
@@ -554,14 +554,16 @@ Specify C++ language in setup.py
 Instead of specifying the language and the sources in the source files, it is
 possible to declare them in the :file:`setup.py` file::
 
-   from setuptools import setup
-   from Cython.Build import cythonize
+    from distutils.core import setup
+
+    from Cython.Build import cythonize
 
-   setup(ext_modules = cythonize(
-              "rect.pyx",                 # our Cython source
-              sources=["Rectangle.cpp"],  # additional source file(s)
-              language="c++",             # generate C++ code
-         ))
+    setup(
+        ext_modules=cythonize(
+            ["rect.pyx", "Rectangle.cpp"],  # our Cython source + additional source file(s)
```

Comment:
I agree that the previous example was wrong, but I'm also suspicious that this works. I don't think it puts the C++ file in the right place, as an additional source file for the `rect.pyx` module. It would probably try to compile a second extension module from it instead.